src/test/registration.test.js

```diff
@@ -74,9 +75,9 @@ describe('Tests for the registration of the users', () => {
       .set('x-access-token', adminToken)
       .send(user[2])
       .end((error, res) => {
-        expect(res).to.have.status([201]);
+        expect(res).to.have.status([403]);
         expect(res.body).to.have.property('status');
-        expect(res.body.status).to.be.equal(201);
+        expect(res.body.status).to.be.equal(403);
         expect(res.body).to.have.property('message');
         expect(res.body.message).to.be.a('string');
```

Comment:
Is there a possibility to assert the error message now that you know it's a string?